From f6f0731b343fbf4741010733c44e6f520ad3592f Mon Sep 17 00:00:00 2001
From: Samuel Dionne-Riel <samuel@dionne-riel.com>
Date: Thu, 30 Dec 2021 14:47:50 -0500
Subject: [PATCH 18/24] aura: sepolicy: remove hint service policy

Why? Not sure, but it fails to compile with LineageOS 18.1

Possibly an Android 12 feature? Or Pixel Experience?
---
 sepolicy/private/razerparts_app.te | 1 -
 1 file changed, 1 deletion(-)

diff --git a/sepolicy/private/razerparts_app.te b/sepolicy/private/razerparts_app.te
index 4e5121b..cfff396 100644
--- a/sepolicy/private/razerparts_app.te
+++ b/sepolicy/private/razerparts_app.te
@@ -15,7 +15,6 @@ allow razerparts_app audio_service:service_manager find;
 allow razerparts_app autofill_service:service_manager find;
 allow razerparts_app content_capture_service:service_manager find;
 allow razerparts_app gpu_service:service_manager find;
-allow razerparts_app hint_service:service_manager find;
 allow razerparts_app surfaceflinger_service:service_manager find;
 allow razerparts_app system_app_data_file:dir remove_name;
 allow razerparts_app system_app_data_file:file { create open rename setattr unlink };
-- 
2.34.0

